optimum/intel/generation/modeling.py

```diff
@@ -88,7 +89,7 @@ def jit_trace(model: PreTrainedModel, task: str, use_cache: bool = False):
     traced_model(**model_inputs)
     traced_model(**model_inputs)
 
-    return traced_model
+    return traced_model, has_position_ids

```

Comment:
I think we should keep `jit_trace` as it is 
```suggestion
    return traced_model

```